test/spinxsk/spinxsk.c

```diff
@@ -1250,6 +1250,7 @@ InitializeQueue(
 {
     HRESULT res;
     QUEUE_CONTEXT *queue;
+    UINT32 apiVersion = RandUlong() % XDP_API_VERSION_LATEST + XDP_API_VERSION_1;
```

Comment:
I can see a possible future where we've gone through the deprecation process for v1 and eventually removed its definition. So, should this simply be `+ 1`?

Reply:
it's fine if it fails - that's kind of the point.